app/Http/Controllers/Api/ConferencesController.php

```diff
@@ -20,12 +20,12 @@ public function index(Request $request)
                 $conferences = EloquentConference::future()->get();
                 break;
             case 'open_cfp':
-                $conferences = EloquentConference::openCfp()->get();
+                $conferences = EloquentConference::whereCfpIsOpen()->get();
```

Comment:
I haven't read the rest of the code, but in the original, "open" and "unclosed" aren't the same.

Open means "you can submit today", unclosed means "the end date of the CFP has not passed, but its start date ALSO may not have passed yet."

Reply:
I realized while implementing the redesigned conference page that there is some overlap between `open` and `unclosed` since all open CFPs are also unclosed.  It seemed more useful to me to change the `unclosed` filter to `future`, which still returns unclosed CFPs but they are constrained to CFPs that have not been opened yet.  This leaves no overlap between `open` and `future`.  Does this seem like a good way to approach it?

In the case of the `Api\ConferencesController` if we go with the above, the `unclosed_cfp` case should change to `future_cfp` and the default should probably be `open_cfp`.

Reply:
It always hurts my brain a bit to think through all this :)

OK, so you have:

- open CFP (you can apply right now)
- future CFP (you could apply in the future but not right now)

With my original, I thought, you may need different filters depending on what your goal is right now.

You may:
- Want to apply right now (open)
- Want to think through what you want to apply to, whether it's open or opening in the future (unclosed)
- Want to look at what you might apply to in the future (future)

That's my original thoughts... now that you've seen my original thoughts, do you think one of my original categories isn't really that common? I'm fine to change it, just want you to know why I had those categories.

Reply:
Thanks for laying this out.  I think it comes down to whether or not there is a motivation to view open and future CFP in a single list that outweighs any potential confusion the unclosed filter might have.  I believe this PR replaces the unclosed filter with the future filter so maybe the path of least resistance is to add the unclosed filter back so users will have all three options. Does that sound like a good path forward to you?